src/client/app/contents/contents.component.html

```diff
@@ -100,7 +100,7 @@ <h3 i18n>Get medicines at the convenience of your home.</h3>
             </div>
         </div>
         <div class="row justify-content-center">
-            <a pageScroll [pageScrollDuration]="500" href="#myHeader"><button type="button" id="myBtn2" style="color: #FFFFFF;" i18n><span class="glyphicon glyphicon-modal-window"></span> Place order</button></a>
+            <!--a pageScroll [pageScrollDuration]="500" href="#myHeader"><button type="button" id="myBtn2" style="color: #FFFFFF;" i18n><span class="glyphicon glyphicon-modal-window"></span> Place order</button></a-->
```

Comment:
Delete the code that is not necessary, yet again. Do not leave the code in comments.